docs/source/events/form-submission.rst

```diff
@@ -27,9 +27,9 @@ Action data
 
 Data about the action and its configuration can be found in ``action``. This contains:
 
-- ``uuid``: A globally unique UUID for this action
+- ``uuid`` (uuid): A globally unique UUID for this action
```

Comment:
maybe double-clarify that this is for the action not the contact taking the action

Reply:
Added a sentence in a new commit.